mlflow/R/mlflow/.run-tests.R

```diff
@@ -10,5 +10,7 @@ keras::install_keras(method = "conda", envname = mlflow:::mlflow_conda_env_name(
 reticulate::conda_install(Sys.getenv("MLFLOW_HOME", "../../../../."), envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
 reticulate::conda_install("xgboost", envname = mlflow:::mlflow_conda_env_name())
 
-devtools::check_built(path = package, error_on = "note", args = "--no-tests")
+# TODO(harupy): Remove `error_on` once the issue below has been fixed:
+# https://stackoverflow.com/questions/63613301/r-cmd-check-note-unable-to-verify-current-time/63616156#63616156
+devtools::check_built(path = package, error_on = "note", args = "--no-tests", error_on = c("never", "error", "warning"))
```

Comment:
Excluded `note` (which was causing the error) from `error_on`.

https://rdrr.io/cran/devtools/man/check.html